homeassistant/components/insteon_plm.py

```diff
@@ -47,13 +57,34 @@
         })
 }, extra=vol.ALLOW_EXTRA)
 
+ADD_ALL_LINK_SCHEMA = vol.Schema({
+    vol.Required(SRV_ALL_LINK_GROUP): vol.Range(min=0, max=255),
+    vol.Required(SRV_ALL_LINK_MODE): vol.In(['C', 'R',
```

Comment:
Don't use single letters, write out the word what it means instead. Also, stick to just lowercase chars